packages/web-api/src/converters/scan-response-converter.ts

```diff
@@ -12,6 +13,9 @@ export class ScanResponseConverter {
 
     public getScanResultResponse(baseUrl: string, apiVersion: string, pageScanResultDocument: OnDemandPageScanResult): ScanResultResponse {
         const runState: OnDemandPageScanRunState = pageScanResultDocument.run.state;
+        const notificationResponse = isNullOrUndefined(pageScanResultDocument.notification)
+            ? {}
+            : { notification: pageScanResultDocument.notification };
```

Comment:
we shouldnt pass the whole error object. the error object may contain some sensitive information. we should only send the error code & a standard error message for the error code.